interface-spec.md

```diff
@@ -95,17 +91,9 @@ Triple is an alias of Quad.
 - `Term .object` the object, which is an IRI, a Literal, a BlankNode or Variable.
 - `Term .graph` the named graph, which is an IRI, DefaultGraph, BlankNode or Variable.
 
-- `Boolean .equals(Quad other)` returns true if and only if the argument is a) of the same type b) has all components equal
-
 **Methods:**
```

Comment:
PR title only mentions removal of toCanonical() and here you remove equals() on Quad